fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/UploadTaskRequest.kt

```diff
@@ -14,8 +14,8 @@ class UploadTaskRequest(request: Request) : TaskRequest(request) {
 
     val BUFFER_SIZE = 1024
 
-    val CRLF = "\\r\\n"
-    val boundary = System.currentTimeMillis().toHexString()
+    val CRLF = "\r\n"
+    val boundary = request.httpHeaders["Content-Type"]!!.split("=", limit=2)[1]
```

Comment:
Sorry for long time not here.

I'm not sure what's the correct behavior when there are no "Content-Type".

According to the RFC of `multipart/form-data`, the parameter `boundary` in "Content-Type" is required to parse the body. If there is no "Content-Type", the body makes completely no sense.